src/tests/java/CodecoolerDAOTest.java

```diff
@@ -0,0 +1,39 @@
+import com.sun.net.httpserver.HttpExchange;
+import controllers.dao.CodecoolerDAO;
+import models.CodecoolerModel;
+import org.junit.jupiter.api.Test;
+import server.codecoolerJavaPages.Store;
+import server.helpers.FormDataParser;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Mockito.*;
+
+
+class CodecoolerDAOTest {
+
+    @Test
+    void testIfCheckingIfCodecoolerHasEnoughMoneyToBuyItemWorksCorrectly() throws IOException {
```

Comment:
testIfCheckingIf.... - it has no sens, isn't better without :
testIfCodecooler... 
?

Reply:
Yes, that's right !

Reply:
But I am not checking if codecooler has enough money to buy item. I am checking if logic responsible for checking that works correctly :( I don't understand.

Reply:
so better it would be:
testIfCodecoolerHasEnoughMoneyMethodWorksCorrectly